doc/classes/Tree.xml

```diff
@@ -619,16 +619,25 @@
 			The arrow icon used when a foldable item is collapsed (for right-to-left layouts).
 		</theme_item>
 		<theme_item name="checked" data_type="icon" type="Texture2D">
-			The check icon to display when the [constant TreeItem.CELL_MODE_CHECK] mode cell is checked.
+			The check icon to display when the [constant TreeItem.CELL_MODE_CHECK] mode cell is checked and set editable with [method TreeItem.set_editable] to [code]true[/code].
+		</theme_item>
+		<theme_item name="checked_disabled" data_type="icon" type="Texture2D">
+			The check icon to display when the [constant TreeItem.CELL_MODE_CHECK] mode cell is checked and set editable with [method TreeItem.set_editable] to [code]false[/code].
```

Comment:
This is a bit confusing, I think it could be simplified to:
```suggestion
			The check icon to display when the [constant TreeItem.CELL_MODE_CHECK] mode cell is checked and non-editable.
```
At most, you could refer to `set_editable()` like this:
```suggestion
			The check icon to display when the [constant TreeItem.CELL_MODE_CHECK] mode cell is checked and non-editable (see [method TreeItem.set_editable]).
```
But I wonder if `set_checked()` also deserves a reference.

Same with other descriptions.